docs/blog/2024-11-15-zmk-studio-mvp-ga.mdx

```diff
@@ -0,0 +1,233 @@
+
+import kinesis from "./assets/2024-11-15-zmk-studio-mvp-ga/kinesis.png";
+import kinesisDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/kinesis-dark-mode.png";
+
+import cannonKeys from "./assets/2024-11-15-zmk-studio-mvp-ga/cannonkeys.png";
+import cannonKeysDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/cannonkeys-dark-mode.png";
+
+import keychron from "./assets/2024-11-15-zmk-studio-mvp-ga/keychron.png";
+import keychronDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/keychron-dark-mode.png";
+
+import littleKeyboards from "./assets/2024-11-15-zmk-studio-mvp-ga/littlekeyboards.avif";
+import littleKeyboardsDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/littlekeyboards-dark-mode.avif";
+
+import keebmaker from "./assets/2024-11-15-zmk-studio-mvp-ga/keebmaker.png";
+import keebmakerDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/keebmaker-dark-mode.png";
+
+import keebio from "./assets/2024-11-15-zmk-studio-mvp-ga/keebio.avif";
+import keebioDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/keebio.avif";
+
+import mode from "./assets/2024-11-15-zmk-studio-mvp-ga/mode.png";
+import modeDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/mode-dark-mode.png";
+
+import phaseByte from "./assets/2024-11-15-zmk-studio-mvp-ga/phasebyte.png";
+import phaseByteDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/phasebyte.png";
+
+import deskHero from "./assets/2024-11-15-zmk-studio-mvp-ga/deskhero.webp";
+import deskHeroDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/deskhero-dark-mode.webp";
+
+import mechlovin from "./assets/2024-11-15-zmk-studio-mvp-ga/mechloving.png";
+import mechlovinDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/mechlovin-dark-mode.png";
+
+import splitkb from "./assets/2024-11-15-zmk-studio-mvp-ga/splitkb.png";
+import splitkbDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/splitkb-dark-mode.png";
+
+import keycapsss from "./assets/2024-11-15-zmk-studio-mvp-ga/keycapsss.png";
+import keycapsssDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/keycapsss-dark-mode.png";
+
+import mekibo from "./assets/2024-11-15-zmk-studio-mvp-ga/mekibo.png";
+import mekiboDarkMode from "./assets/2024-11-15-zmk-studio-mvp-ga/mekibo-dark-mode.png";
+
+I'm delighted to announce that after many months of work, the ZMK Studio MVP has reached General Availability (GA) status. I'm very excited with the feature set and foundation we've gotten into this first release, and equally excited for the upcoming work to add even more features.
+
+ZMK Studio provides runtime keymap functionality for ZMK powered devices. It is a combination of major firmware enhancements paired with the new frontend application for interacting with you keyboard to make changes.
+
+The app is available for compatible browsers, and as an installable desktop application for Linux, macOS, and Windows.
+
+![ZMK Studio Application](./assets/2024-11-15-zmk-studio-mvp-ga/zmk-studio-screenshot.png)
```

Comment:
Is it possible to have a light mode screenshot when the website is in light mode?

Reply:
Done.